snippets/change_notifier_provider.code-snippets

```diff
@@ -12,11 +12,11 @@
   "Change Notifier Provider with Family": {
     "scope": "dart",
     "prefix": "changeNotifierProviderFamily",
-    "description": "Create a Change otifierProvider with Family modifier",
+    "description": "Create a Change Notifier Provider with Family modifier",
```

Comment:
The description of these classes should be one word so "ChangeNotifierProvider". The same with the others 😊